app/src/main/java/org/hackillinois/android/model/mentor/MentorModel.kt

```diff
@@ -0,0 +1,7 @@
+package org.hackillinois.android.model.mentor
+
+data class MentorModel(
```

Comment:
I'm not up to speed with this PR, but I'm thinking this should be changed to ProjectModel. In the backend, the service is called `project` instead of `mentor`, so it would make sense to match that naming.

Reply:
Agreed. We should also rename the MentorFragment to ProjectFragment, as Brian said in the other PR.

Reply:
Sure, will do. Thanks for the comments, I appreciate it!